doc/ranger.pod

```diff
@@ -302,6 +302,20 @@ The macro %space expands to a space character. You can use it to add spaces to
 the end of a command when needed, while preventing editors to strip spaces off
 the end of the line automatically.
 
+The macros %any and %any_path can be used when binding keys to
+commands. The macro %any will exapnd to the key pressed in the place of
```

Comment:
Typo "exapnd". Maybe rephrase something along the lines of "the placeholder `<any>` will be replaced with the key...". Same thing for the next sentence.